Add strings for translation

urbalazs opened the pull request:
Hello @mhall119 

I tried to solve all translation issue with this PR as you mentioned in https://github.com/GetTogetherComm/GetTogether/issues/271

Please review it and fix, if needed. Can you update the language files (*.po) after merging this? There are some new string from translation with this PR and with my previous one. Thanks!

thiagoferreiraw commented on events/models/profiles.py:
```diff
@@ -705,7 +713,11 @@ def save(self, *args, **kwargs):
     category = models.ForeignKey(
-        Category, on_delete=models.CASCADE, null=False, blank=False
+        Category,
+        verbose_name = "Category"
```
There's a missing a comma here  :) 
```suggestion
        verbose_name = "Category",
```

thiagoferreiraw commented:
@urbalazs , also, since you changed some models files, you'll need to run the command `python manage.py makemigrations` and commit the generated files.

urbalazs commented:
@thiagoferreiraw I fixed the missing comma with forced push. About the file generation: I have no developer environment set, I just changed the source with a basic text editor. Can I ask you (or someone else) to run the command `python manage.py makemigrations` and commit the generated files? Thanks!
